pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -322,6 +333,25 @@ func (ctrl *PersistentVolumeController) emitEventForUnboundDelayBindingClaim(cla
 			message = fmt.Sprintf("waiting for pod %s to be scheduled", podNames[0])
 		}
 	}
+	// we already emitted an event and we have the condition set, do nothing
+	if c := findStatusCondition(claim.Status.Conditions, v1.PersistentVolumeClaimWaitingForConsumer); c != nil && c.Status == v1.ConditionTrue {
+		return nil
+	}
+
+	// this is the first time we see waiting for consumer, set the condition and emit event
+	newClaim := claim.DeepCopy()
+	newClaim.Status.Conditions = append(newClaim.Status.Conditions, v1.PersistentVolumeClaimCondition{
+		Type:               v1.PersistentVolumeClaimWaitingForConsumer,
+		Status:             v1.ConditionTrue,
+		LastProbeTime:      metav1.Now(),
```

Comment:
this is a little controversial... we should probably update this every time the controller see this PVC, but it might generate more API traffic. Not sure how much useful this is.

Reply:
Is it possible to set only `LastTranstionTime` and have `LastProbeTime` nil?

Reply:
@jsafrane yes

Reply:
fixed